src/odemis/gui/cont/acquisition.py

```diff
@@ -850,12 +829,16 @@ def _on_zstack(self):
         self._zlevels = {}
         for s in self._acquiStreams.value:
             if isinstance(s, FluoStream):
+                if not self._tab_data.main.focus:
+                    raise LookupError("Focuser was not found in the tab data")
```

Comment:
Having a FM acquisition without focus would be really odd. Actually, I'll make a PR for a extra check at init that will complain if this component is missing. So no need to check for it.

Reply:
Please, drop these two lines. There is now a check that "focus" exists.